### PR TITLE
KtCleanup replaced `is` with equalTo 

### PR DESCRIPTION
**Pull request by criticalAY**
## Pull Request template
KotlinCleanup

## Purpose / Description
replaced is with equalTo function

## Fixes
Fixes _Link to the issues._

## Approach
_How does this change address the problem?_

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration (SDK version(s), emulator or physical, etc)

## Learning (optional, can help others)
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by criticalAY** [comment]
```
 java.lang.AssertionError: 
    Expected: <a value equal to or greater than <1596783600L>>
         but: was <1596784200L>
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
        at com.ichi2.libanki.sched.SchedTest.test_suspendV1(SchedTest.kt:753)
```

Should we go for 30L? after replacing `is` with `equalTo`  does that hampers it somehow?

**Comment by criticalAY** [comment]
I would also like to know if we can replace 
``` 
val t = Math.round((c.due - TimeManager.time.intTime()).toFloat()).toLong()
```
with 
```
val t = (c.due - TimeManager.time.intTime()).toFloat().roundToLong()
```
seems relevant to me though

**Review by david-allison — COMMENTED** [review]
From the previous diff, I don't believe this fixes all the issues defined by `@KotlinCleanup("is -> equalTo")`

File: AnkiDroid/src/test/java/com/ichi2/libanki/sched/SchedTest.kt [hunk 1]
```diff
@@ -499,8 +498,8 @@ class SchedTest : RobolectricTest() {
         assertEquals(1, c.ivl)
         // but because it's in the learn queue, its current due time should be in
         // the future
-        assertThat(c.due, `is`(greaterThanOrEqualTo(TimeManager.time.intTime())))
-        assertThat(c.due - TimeManager.time.intTime(), `is`(greaterThan(118L)))
+        assertThat(c.due, equalTo(greaterThanOrEqualTo(TimeManager.time.intTime())))
```
[the remark this answers is not in the document]
Comment (criticalAY) [reply AnkiDroid/src/test/java/com/ichi2/libanki/sched/SchedTest.kt]: tried that but still failed

**Comment by criticalAY** [comment]
you were right I must have missed some line so the test failed earlier, Thanks!

**Comment by criticalAY** [comment]
@BrayanDSO may I ask something? like the PRs are getting merged after one approval I noticed it on two of my PRs? is there some specific reason ?

**Comment by BrayanDSO** [comment]
On the [reviewing guide](https://github.com/ankidroid/Anki-Android/wiki/Reviewing-and-submitting#approving-change):

> If a reviewer believes that their judgement is enough, it is perfectly fine to merge the code without a second review.

And there have been a lot of exceendingly simple cleanup PRs lately, although I'd rather if the focus was on solving issues or implementing new features than cleaning code that has a high change of being refactored or removed on the future

**Comment by mikehardy** [comment]
Yeah +1 on one-approval merges for PRs that are just obviously correct. It saves more "reviewing power" for the hard PRs, and there are plenty of those to review...

**Comment by criticalAY** [comment]
gotchu. I already planned to divert my focus from cleanups to other issues though because I see a lot of them coming recently

**Pull request merged**